modules/runner/src/runner.ts

```diff
@@ -75,63 +141,74 @@ export class Runner {
   }
 
   public start(): void {
+    const d = debug(`${DebugPrefix}:start`);
+
     this.stop();
     d('runner:start', `interval is ${this.pollIntervalMs}`);
-    this.interval = setInterval(() => this.pollSafely(), this.pollIntervalMs);
+    this.pollInterval = setInterval(
+      () => this.pollSafely(),
+      this.pollIntervalMs,
+    );
```

Comment:
question: is there a difference between adding a wrapper function and just doing this?
```suggestion
    this.pollInterval = setInterval(this.pollSafely,
      this.pollIntervalMs,
    );
```

Reply:
Yes, the latter generates an eslint warning for `this`:

Reply:
Updated in eb87705f50ac13057e3c0d2b949fb3cf72c7e0a2